ffv1.md

```diff
@@ -526,7 +526,7 @@ Figure: A pseudo-code description of the contexts of Range Non Binary Values. {#
 
 #### Initial Values for the Context Model
 
-At keyframes all Range coder state variables are set to their initial state.
+When `keyframe` (see (#frame)) value is 0, all Range coder state variables are set to their initial state.
```

Comment:
> When the `keyframe` value is 0 (see (#frame)),
 

and below

Reply:
All issues fixed but this one "and below", I don't get the expected meaning.